src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/DirectoryBasedStoreProvider.cs

```diff
@@ -84,11 +80,6 @@ internal DirectoryBasedStoreProvider(string storeName, OpenFlags openFlags)
 
         public void Dispose()
```

Comment:
Does IStorePal still need to be IDisposable?  I've not looked to see if other implementations have code in it.